src/core/requests/RoutesNames.js

```diff
@@ -0,0 +1,9 @@
+
+
+const routesNames = {
```

Comment:
To powinno się znaleść w pliku routes.js, być przez niego wykorzystywane i stamtąd eksportowane. Teraz mamy duplikację informacji o dostępnych routach